src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package tasks;
+
+public class Deadline extends Task {
+
+    public String deadline;
+    //public static String taskType = "D";
```

Comment:
avoid leaving in commented-out code. it can reduce readability

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#delete-dead-code